[Docs] Snapshot config explanation. (#6439)
blakeblackshear/frigate: docs/docs/configuration/snapshots.md

---

Frigate can save a snapshot image to `/media/frigate/clips` for each event named as `<camera>-<id>.jpg`.

> Camera specific snapshot can be configured in `config.yml` under  `cameras -> camera -> mqtt` over at [config file](https://docs.frigate.video/configuration/)

Reviewer: ```suggestion
Object specific snapshots for each camera can be configured in the [config file](https://docs.frigate.video/configuration/) under  `cameras -> your_camera -> mqtt`
```

Reviewer: Not all config files are named the same, also want to be clear it is not the same snapshot that is referenced above in that file
